frontend/wallet/src/render/DownloadLogic.re

```diff
@@ -1,3 +1,4 @@
+/** Bindings **/
```

Comment:
Aren't these bindings still duplicates of the ones in Bindings.re

Reply:
These are left over from the old downloader logic. I can remove them if you like.